sections/home-section/landing/index.js

```diff
@@ -1,10 +1,23 @@
+import { useState, useEffect } from 'react'
 import Router from 'next/router'
 import * as S from './Landing.styled'
 import Button from '../../../components/button'
 import TabBar from '../../../components/tab-bar'
 
-const Landing = () => (
-	<S.LandingWrapper>
+const Landing = () => {
+	const [current, setCurrent] = useState(0)
+	const landingGifUrls = [
+		'/static/gifs/financial-business-chart.gif',
+		'/static/gifs/new-york-cityscape.gif'
+	]
+
+	useEffect(() => {
+		const next = (current + 1) % landingGifUrls.length
+		const id = setTimeout(() => setCurrent(next), 5990)
```

Comment:
let's stay away from having magic numbers and move `5990` to a constant at the top of the file